tests/store/test_status_samples_filters.py

```diff
@@ -0,0 +1,49 @@
+from alchy import Query
+
+from cg.constants.subject import Gender
+from cg.store.status_sample_filters import (
+    get_samples_with_loqusdb_id,
+    get_samples_without_loqusdb_id,
+)
+
+
+def test_get_samples_with_loqusdb_id(helpers, store, sample_store):
+    """Test sample extraction with Loqusdb ID."""
+
+    # GIVEN a sample observations that has been uploaded to Loqusdb
+    case = helpers.add_case(store)
+    sample = helpers.add_sample(store, loqusdb_id="uploaded_to_loqusdb")
+    sample_not_uploaded = helpers.add_sample(store, name="test")
```

Comment:
should we make the helper use the sample_id fixture? It would be more consistent

Reply:
It's already on place, we can provide it as a **kwargs. I will use the sample_id fixture :)